RxCocoa/RxCocoa/iOS/UITextField+Rx.swift

```diff
@@ -12,21 +12,31 @@ import UIKit
 
 extension UITextField {
     public func rx_text() -> Observable<String> {
+        return observableForControlEvent(.EditingChanged)
+    }
+
+    public func rx_endOnExit() -> Observable<String> {
```

Comment:
Well, don't forget that I added an rx_ in UIControl to handle all the events. It's just that the UIControl method doesn't return the value contained in the string.